src/com/ka/spreadsheet/diff/StdoutSpreadSheetDiffCallback.java

```diff
@@ -56,6 +75,13 @@ public void reportExtraCell(boolean inFirstSpreadSheet, CellPos c) {
 
   @Override
   public void reportDiffCell(CellPos c1, CellPos c2) {
+    assert (c1.getRowIndex() == c2.getRowIndex())
+      && (c1.getColumnIndex() == c2.getColumnIndex()) : "Cells are not at the same position. Cell 1="
+      + c1.getCellPosition() + ", cell 2=" + c2.getCellPosition();
+    assert previousCell == null || c1.compareCellPositions(previousCell) >= 0 :
```

Comment:
this can be > 0 instead of >= 0

Reply:
Fixed here and in UnifiedDiffSpreadSheetDiffCallback, and also in both versions of `reportExtraCell()`